qsirecon/utils/bids.py

```diff
@@ -185,7 +186,7 @@ def write_derivative_description(bids_dir, deriv_dir, dataset_links=None):
     if "QSIRECON_DOCKER_TAG" in os.environ:
         dset_desc["GeneratedBy"][0]["Container"] = {
             "Type": "docker",
-            "Tag": f"nipreps/fmriprep:{os.environ['QSIRECON_DOCKER_TAG']}",
+            "Tag": f"pennbbl/qsirecon:{os.environ['QSIRECON_DOCKER_TAG']}",
```

Comment:
I think qsirecon is being published to pennlinc instead of pennbbl

Reply:
Ah OK. @mattcieslak is it pennlinc? I will fix

Reply:
I forgot this is already built so I can check for myself on docker hub. Fixed

Reply:
Yes, we're moving these to pennlinc as part of The Great Split